defaults/main.yml

```diff
@@ -4,10 +4,17 @@ rstudio_workbench_version: "2021.09.0-351.pro6"
 rstudio_workbench_install: []
 
 rstudio_workbench_www_port: 8787
-rstudio_workbench_config_override: ""
 
-rstudio_workbench_http_override: |
-    www-port={{ rstudio_workbench_www_port }}
+rstudio_workbench_rserver_config_override: |
+  www-port={{ rstudio_workbench_www_port }}
+  server-health-check-enabled=1
```

Comment:
If you'll hardcode everything in single variable then you won't be able to do things like `wait_for` or more complex logic in a sane way, since you won't have those variables defined.

Reply:
We discussed that some variables like port should be defined as seperate vars and then included in the config file. Updated in newest commit.